src/conda_package_handling/interface.py

```diff
@@ -15,7 +14,7 @@ def extract(fn, dest_dir, **kw):  # pragma: no cover
 
     @staticmethod
     @abc.abstractmethod
-    def create(prefix, file_list, out_fn, out_folder=os.getcwd(), **kw):  # pragma: no cover
+    def create(prefix, file_list, out_fn, out_folder: str, **kw):  # pragma: no cover
```

Comment:
Is this supposed to be like:

```suggestion
    def create(prefix, file_list, out_fn, out_folder: str | None = None, **kw):  # pragma: no cover
```